docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2.4'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: app
+    env_file: .env
+    image: app:latest
+    ports:
+      - 3000:3000
+      - 7723:7723
```

Comment:
I think being this is a public repository, we don't have tp expose our internal port convention, just use `3000` for now